roachtest: sqlsmith/setup=seed/setting=no-ddl failed

roachtest.sqlsmith/setup=seed/setting=no-ddl [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3658629&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3658629&tab=artifacts#/sqlsmith/setup=seed/setting=no-ddl) on master @ [db229cad354c2776ea35c9dc4f78e6ce66b9437c](https://github.com/cockroachdb/cockroach/commits/db229cad354c2776ea35c9dc4f78e6ce66b9437c):


```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/sqlsmith/setup=seed/setting=no-ddl/run_1
	sqlsmith.go:224,sqlsmith.go:257,test_runner.go:777: error: pq: internal error: crdb_internal.reset_index_usage_stats(): index usage stats controller not set
		stmt:
		SELECT
			crdb_internal.reset_index_usage_stats()::BOOL AS col_3262, '\xa318b179':::BYTES AS col_3263
		FROM
			defaultdb.public.seed@seed__int8__float8__date_idx AS tab_1358
		WHERE
			tab_1358._bool
		ORDER BY
			tab_1358._enum ASC, tab_1358._inet
		LIMIT
			90:::INT8;
```
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

 | 

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlsmith/setup=seed/setting=no-ddl.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/internal/sqlsmith/schema.go
+++ b/pkg/internal/sqlsmith/schema.go
@@ -493,6 +493,7 @@ var functions = func() map[tree.FunctionClass]map[oid.Oid][]function {
 			"crdb_internal.unsafe_",
 			"crdb_internal.create_join_token",
 			"crdb_internal.reset_multi_region_zone_configs_for_database",
+			"crdb_internal.reset_index_usage_stats",
 		} {
 			skip = skip || strings.Contains(def.Name, substr)
 		}

